brayns/io/XYZBLoader.cpp

```diff
@@ -39,12 +39,10 @@ std::set<std::string> XYZBLoader::getSupportedDataTypes()
 }
 
 void XYZBLoader::importFromBlob(Blob&& blob, Scene& scene,
+                                const size_t index BRAYNS_UNUSED,
                                 const Matrix4f& transformation,
-                                size_t materialID)
+                                const size_t defaultMaterialId BRAYNS_UNUSED)
 {
-    if (materialID == NO_MATERIAL)
```

Comment:
you sure? if it is NO_MATERIAL, it's infinity, and you see no geometry :) Probably you changed that behavior.